lib/travis/build/appliances/apt_get_update.rb

```diff
@@ -5,13 +5,33 @@ module Build
     module Appliances
       class AptGetUpdate < Base
         def apply
+          retry_under = Travis::Build.config.apt.retries.percentage.to_i || 0 # if under this, we add retry config
```

Comment:
What is the intent with this rollout percentage?

Reply:
"X % of all jobs will use retries set by the appliances"

Reply:
@BanzaiMan I understand the mechanics, but what is your intent regarding the speed of rollout, how we're tracking this, why not all at once, etc?

Reply:
We don't have a good handle on how often the issue is manifesting. Only user feedback. We won't know how effective this is, and it seems prudent to me that we do not go to the deep end without testing it on a smaller scale.

We can drop the appliances idea, until the opt-in users confirm better result, and go all in, if the scheduling the rollout seems cumbersome.